ILSpy/AssemblyTree/AssemblyTreeModel.cs

```diff
@@ -767,6 +767,7 @@ private void TreeView_SelectionChanged()
 				}
 				else
 				{
+					ContextMenuProvider.ContextMenuClosed -= ContextMenuClosed;
```

Comment:
Please add a short comment about the intent of the trick (to avoid anyone scratching their head years later on why or even worse actually removing something that looks out of place)

Reply:
This is just a common pattern to avoid duplicate event registration and execution.
It is not the primary fix, only avoids side effects.